src/lib/object-class.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the object class for any value.
+ *
+ * @param {*} [value] - A value whose object class should be determined.
+ *
+ * @example
+ * const isNull = objectClass(null) === 'null'
+ * const isPOJO = objectClass({}) === 'object'
+ *
+ * @returns {String}
+ */
+const objectClass = (value) => /\[\w+ (\w+)]/.exec(Object.prototype.toString.call(value))[1].toLowerCase()
```

Comment:
I mostly use `({}).toString.call` for giving the best info to the user concerning array vs null vs pojo.  Will consider your comments if/when merging.